api/translator/src/main/java/com/project/translator/adapter/out/persistence/LanguageEntity.java

```diff
@@ -0,0 +1,54 @@
+package com.project.translator.adapter.out.persistence;
+
+import jakarta.persistence.*;
+import lombok.*;
```

Comment:
Prośba o nie zawijanie importów do * - nie wiadomo ile rzeczy się zaciąga. Uwaga do wszystkich * we wszystkich plikach

Reply:
poprawione